test/crypto_SUITE.erl

```diff
@@ -1,71 +0,0 @@
--module(crypto_SUITE).
```

Comment:
Why was this removed?

Reply:
Because the SUITE was moved into the unit tests inside the .erl file a while ago. We just never noticed